bundles/org.openhab.binding.freeboxos/README.md

```diff
@@ -0,0 +1,177 @@
+# Freebox Binding

```

Comment:
I guess it should be:
```suggestion
# FreeboxOS Binding
```

? But please see latest PR comments before changing anything.